uiowa/uiowa
# Added back to top link functionality for Grad site

# How to test
1. Checkout branch and run `blt ds --site=grad.uiowa.edu`
2. Run `blt frontend`
3. Navigate to http://grad.local.drupal.uiowa.edu/academics/thesis-and-dissertation/new-author-deposit-agreement and make sure there's a top button on the right side of the page
3. Go to http://grad.local.drupal.uiowa.edu/admin/appearance/settings/uids_base and make sure there's an option to not display the scroll to top button
4. Check the checkbox stating you want to hide the button link
5. navigate back to http://grad.local.drupal.uiowa.edu/academics/thesis-and-dissertation/new-author-deposit-agreement and make sure the button does not display
